DetectorDescription/Detectors/src/SimpleCone_geo.cxx

```diff
@@ -0,0 +1,51 @@
+// $Id: $
+//====================================================================
+//  AIDA Detector description implementation for FCC
+//-------------------------------------------------------------------- 
+// Simple cone using dimensions to be used to define cone composed of 1 single material
+//
+// @author Clement Helsens
+//====================================================================
+#include "DD4hep/DetFactoryHelper.h"
+
+using namespace std;
+using namespace DD4hep;
+using namespace DD4hep::Geometry;
```

Comment:
Can you remove these `using namespace` statements? If you want to avoid typing the full names please do it like e.g. here: https://github.com/HEP-FCC/FCCSW/blob/master/DetectorDescription/Detectors/src/HCalBarrel_geo.cxx#L7